cli/pack/common.go

```diff
@@ -106,7 +106,7 @@ func prepareBundle(cmdCtx *cmdcontext.CmdCtx, packCtx *PackCtx,
 	if opts.Modules != nil && opts.Modules.Directory != "" {
 		err = copy.Copy(opts.Modules.Directory, packageEnvModulesPath)
 		if err != nil {
-			return "", err
+			log.Warnf("Failed to copy modules from %s: %s", opts.Modules.Directory, err)
```

Comment:
Right.

Reply:
Oh, that's nice. Thank you.

But what exactly stored in `modules`?

Reply:
See https://github.com/tarantool/tt#external-modules.

Reply:
So any module that is installed on my system will be added to RPM package (if I use default global settings)? What's about tt itself?

Reply:
tt will not be included in the result package if you specify the `--without-binaries` flag or use system environment. 

> So any module that is installed on my system will be added to RPM package (if I use default global settings)?

Yes, but using the global config is not recommended.

Reply:
> tt will not be included in the result package if you specify the --without-binaries

What's about modules? It's a bit weird that we add additional modules for tt by default but skip tt itself.

> Yes, but using the global config is not recommended.

Why not? The idea of having the same default config in all of my directories of several levels scares me. Especially since I mostly work on Lua rocks, not applications.

Reply:
> > tt will not be included in the result package if you specify the --without-binaries
> 
> What's about modules? It's a bit weird that we add additional modules for tt by default but skip tt itself.
> 

https://github.com/tarantool/tt/issues/289

> > Yes, but using the global config is not recommended.
> 

This is a valid and popular scenario in CI.

> Why not? The idea of having the same default config in all of my directories on several levels scares me. Especially since I mostly work on Lua rocks, not applications.